tests/testthat/test_loo_moment_matching.R

```diff
@@ -0,0 +1,303 @@
+library(loo)
```

Comment:
do we have tests somehwere that the diagnostic pareto k values are actually updated while the leverage values remain the same?

Reply:
This happens in `test_that("mmloo.default works", {`. I added an explicit tests to check this.